src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -170,7 +169,7 @@ void SubgraphBaseTest::compare(const std::vector<ov::Tensor>& expected,
                     inputNode = nextNodePtr;
                 }
             }
-            auto it = compareMap.find(inputNode->get_type_info());
+            auto it = compareMap.size() == 1 ? compareMap.begin() : compareMap.find(inputNode->get_type_info());
```

Comment:
HARDCODE!!!! What is that?! I have left the comment connected to these method above. Let's setup short call